frontend/src/router.ts

```diff
@@ -29,11 +31,26 @@ const router = createRouter({
       name: ROUTES.CLIENT.name,
       component: ClientView,
     },
+    {
+      path: `${ROUTES.CLIENT.path}/new`,
```

Comment:
Can we make the path different?

The case that a clientID is "new" is low, but route checking might get confusing anyway.

maybe something like the below works?

```suggestion
      path: `${ROUTES.CLIENT.path}-new`,
      // or 
      path: `${ROUTES.CLIENT_NEW.path}`,
```